### DISCUSSION TITLE
PAGELLE MODELLO MINISTERIALI

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#274] [Idee e proposte]
Buonasera , oltre alle  classico modelle di  pagelle (quelle già presenti nel sistema) , sarebbe possibile introdurre anche i modelli ministeriali? Riporto alcuni esempi :

[Scheda-di-Valutazione-Primaria.pdf](https://github.com/trinko/giuaschool/files/8886928/Scheda-di-Valutazione-Primaria.pdf)

**Comment by trinko** [comment]
Non per questo anno scolastico.
La inserisco nelle richieste.
